docs/docs/exceptions.md

```diff
@@ -119,3 +119,27 @@ use a custom exception handler that doesn't fall through, for example:
 def black_hole(job, *exc_info):
     return False
 ```
+
+## Workhorse Termination Handler
```

Comment:
Can you add the text `_New in version 1.13.0_`? An example is here: https://github.com/rq/rq/blob/a02ad29cef71712376aa0b284b4cb2e3aea576fd/docs/docs/index.md?plain=1#L239